hllplus/hllplus_test.go

```diff
@@ -93,9 +128,41 @@ var _ = Describe("HLL", func() {
 		Entry("p=24", 24, 149988),
 	)
 
+	DescribeTable("estimate sparse (100k unique + 2x50k)",
+		func(p int, exp int) {
+			subject, _ = hllplus.New(uint8(p-5), uint8(p))
+			for i := 0; i < 100_000; i++ {
+				subject.Add(rnd.Uint64())
+			}
+			for i := 0; i < 50_000; i++ {
+				h := rnd.Uint64()
+				subject.Add(h)
+				subject.Add(h)
+			}
+			Expect(subject.IsSparse()).To(BeTrue())
+			Expect(subject.Estimate()).To(Equal(int64(exp)))
+		},
+		Entry("p=23", 23, 150823),
+		Entry("p=24", 24, 150144),
+		Entry("p=25", 25, 150012),
+	)
+
+	FIt("should normalize", func() {
```

Comment:
This is the test that is currently failing/needs fixing, pls ignore the failing tests in the root package for now